### PR TITLE
Get server command

### PR DESCRIPTION
**Pull request by danielmlozano**
`spinupwp servers:get <server_id>`

### CODE REVIEW

File: app/Commands/Servers/GetCommand.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+        return [
+            'ID'            => $server->id,
+            'Name'          => $server->name,
+            'Provider name' => $server->provider_name,
+            'IP Address'    => $server->ip_address,
+            'Ubuntu'        => $server->ubuntu_version,
+            'Database'      => $server->database['server'],
+            'SSH port'      => $server->ssh_port,
+        ];
```
Comment (A5hleyRich) [inline app/Commands/Servers/GetCommand.php:33]: This needs to output everything returned from the API.

File: app/Commands/Servers/GetCommand.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        $server = $this->spinupwp->servers->get($serverId);
+
+        if ($this->displayFormat() === 'json') {
+            return $server;
+        }
+
+        return [
+            'ID'            => $server->id,
+            'Name'          => $server->name,
+            'Provider name' => $server->provider_name,
```
Comment (A5hleyRich) [inline app/Commands/Servers/GetCommand.php:28]: Provider **N**ame

File: app/Commands/Servers/GetCommand.php [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            'IP Address'    => $server->ip_address,
+            'Ubuntu'        => $server->ubuntu_version,
+            'Database'      => $server->database['server'],
+            'SSH port'      => $server->ssh_port,
```
Comment (A5hleyRich) [inline app/Commands/Servers/GetCommand.php:32]: SSH **P**ort

**Review by A5hleyRich — CHANGES_REQUESTED** [review]

**Comment by A5hleyRich** [comment]
@danielmlozano as discussed in Slack, can we also change the output to match the list commands? The table should have a border and the left column keys should be green.

**Comment by danielmlozano** [comment]
@A5hleyRich Done. The only thing that I'm not sure about is that the SSH and Git keys are too large for the table.

**Comment by danielmlozano** [comment]
@A5hleyRich added the `setColumnMaxWidth` method.

**Comment by A5hleyRich** [comment]
💯

**Pull request merged**